src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes_v2/style.scss

```diff
@@ -0,0 +1,94 @@
+.baseNode {
+  width: 200px;
+  background-color: #ededed;
+  border: 1px solid #dedede;
+  border-radius: 15px;
+
+  :global(.port) {
+    background: rgba(0, 0, 0, 0.2);
+
+    &:global(.selected) {
+      background: rgba(0, 0, 0, 0.5);
+    }
+  }
+
+  .header {
+    background: #a1c6e8;
+    color: #ffffff;
+    height: 20px;
+    padding: 2px 10px;
+    font-size: 12px;
+    border-radius: 15px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .content {
+    padding: 4px;
+    word-wrap: break-word;
+    text-overflow: ellipsis;
+    max-height: 78px;
+    overflow-y: scroll;
+    font-size: 12px;
+  }
+
+  .ports {
+    .in {
+      position: absolute;
+      bottom: 0px;
+      left: -8px;
+    }
+    .out {
+      position: absolute;
+      bottom: 0px;
+      right: -8px;
+    }
+  }
+}
+
+.highlightedNode {
+  box-shadow: 0 0 0 4pt #ffde33;
+}
+
+.nodeSaySomething {
+  .header {
+    background: #a1c6e8;
+    color: #ffffff;
+  }
+}
+
+.nodeExecute {
+  .header {
+    background: #d5a5bc;
+    color: #ffffff;
+  }
+}
+
+.nodeListen {
+  .header {
+    background: #ffe699;
+    color: #000000;
+  }
+}
+
+.nodeRouter {
+  .header {
+    background: #a1c4c9;
+    color: #000000;
+  }
+
+  .content {
+    overflow: auto;
+    max-height: 250px;
+  }
+}
+
+.section-next .item :global(.port) {
+  background: rgba(0, 0, 0, 0.2);
+
+  &:global(.selected) {
+    background: rgba(0, 0, 0, 0.5);
+  }
+}
+
+/* OLD SHIT */
```

Comment:
stuff* :)

Reply:
Haha, should have been removed completely :p